src/main/java/lotto/LottoGame.java

```diff
@@ -0,0 +1,156 @@
+package lotto;
+
+import camp.nextstep.edu.missionutils.Console;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class LottoGame {
+	//
+    private static final int LOTTO_PRICE = 1000;
+
+    //method
+    public void start() {
+    	//구성요소 
+        int money = getMoney();
+        int lottoCount = getLottoCount(money);
+        List<Lotto> boughtLottos = buyLottos(lottoCount);
+        printLottos(boughtLottos);
+        
+        // 사용자가 당첨 번호와 보너스 번호 입력
+        List<Integer> winningNumbers = getWinningNumbers();
+        int bonusNumber = getBonusNumber(winningNumbers);
+
+        // Lotto 클래스에서 자동 생성된 번호와 비교
+        calculateResults(boughtLottos, winningNumbers, bonusNumber);
+    }
+
+    //Input
+    //입금
+    private int getMoney() {
+        System.out.println("구입금액을 입력해 주세요.");
+        String input = Console.readLine();
+        return validateMoney(input);
+    }
+    //잘못된 입금 시 에러 처리
+    private int validateMoney(String input) {
```

Comment:
무언가를 검사하는 메소드에서 String타입을 int타입의 Money를 반환하는 것을 올바르지 않은 것 같습니다.
메소드를 분리해주세요
```suggestion
    private void validateMoney(String input) {
```